components/FinancialSummary/ExpenseBreakdown.js

```diff
@@ -22,7 +23,7 @@ export default function ExpenseBreakdown() {
                             <div className="text-darkGunMetal">
                                 <div className="flex flex-col items-center">
                                     <img src="/img/illustrations/MentorshipProgram.webp" alt="Mentorship Program" className="w-1/5 h-auto object-cover rounded-md m-1" />
-                                    <h2 className="text-2xl font-semibold my-2 text-center text-darkGunMetal">Mentorship Program</h2>
+                                    <h2 className="text-2xl font-semibold my-2 text-center text-darkGunMetal hover:underline cursor-pointer"><Link href="https://www.asyncapi.com/blog/beyond-boundaries">Mentorship Program</Link></h2>
```

Comment:
Links should be added to the whole Card, not only to the Card Title.

Reply:
@akshatnema added the suggested changes